### PR TITLE
Add doc about how to perform automated recovery

### PR DESCRIPTION
**Pull request by evanphx**
This document help users configure their systems to perform an automated recovery operation, which is needed to achieve low MTTR values.

### CODE REVIEW

**Comment by KFishner** [comment]
@evanphx i was going to write a review but it was more direct to just make the changes - hope you dont mind! I added an introduction paragraph and changed the ordering to be more linear. let me know what you think

File: content/source/docs/enterprise/private/automated-recovery.html.md [hunk 1]
```diff
@@ -6,35 +6,58 @@ sidebar_current: "docs-enterprise2-private-automated-recovery"
...
 
-## Version Checking
+Snapshots are taken on the Private TFE instance. That instance can
+have two types of data on it:
+
+- Terraform Enterprise application data: The core product data such as
+run history, configuration history, state history. This data
+changes frequently.
+- Terraform Enterprise instance configuration data: The data used
+to configure Private TFE itself such as SMTP credentials, admin
```
Comment (evanphx) [inline content/source/docs/enterprise/private/automated-recovery.html.md]: This isn't really true at present, SMTP creds and admin creds are not stored outside of the database.

**Comment by evanphx** [comment]
@KFishner No problem but one comment above about the "configuration" data, which does not include SMTP creds, admin creds, etc.

**Comment by KFishner** [comment]
@evanphx fixed!

**Pull request merged**